timeseries/src/main/scala/com/criteo/cuttle/timeseries/TimeSeriesApp.scala

```diff
@@ -760,6 +760,8 @@ private[timeseries] case class TimeSeriesApp(project: CuttleProject, executor: E
           case Done(_) => true
           case _ => false
         }}
+        .foldLeft(IntervalMap.empty[Instant, Unit])((acc, elt) => acc.update(elt._1, ()))
```

Comment:
This probably works, but you can also write

```(scala)
    val successfulIntervalMaps = jobStates
        .filter(s => s._1.id == jobId)
        .values
        .flatMap(m => m.map({
          case Done(_) => true
          case _ => false
        }).toList)
```
taking advantage of the fact IntervalMap[Instant, _] is a Functor, we can map from IntervalMap[Instant, JobState] to IntervalMap[Instant, Boolean] just by providing the function 

```
jobState => jobState match {
  case Done(_) => true
  case _            => false
}
```